### DISCUSSION TITLE
Cannot Change the homepage

### DISCUSSION THREAD

**Discussion by gt-yj** [LekoArts/gatsby-themes#601] [Questions]
After Deploying the theme on gatsbyjs cloud, I clone'ed the repo in my pc to make changes. After I Looked, I can't find the page which is rendering the homepage. I need to change the homepage and remove the template paragraphs. 

Which file renders the homepage and where it is located?

**Comment by LekoArts** [comment]
You'd need to be more specific on what theme you're using, but every theme has instructions in its README on how you can change it.
